src/config/Globals.js

```diff
@@ -101,6 +101,8 @@ export default {
   CONSTANTS: {
     HEADER_HEIGHT: 70,
     LEFT_BAR_WIDTH: 260,
+    COLLAPSABLE_LEFT_BAR_WIDTH:90,
+    COLLAPSABLE_RIGHT_BAR_WIDTH:100,
 
```

Comment:
We also need to cancel the margin of all modules when sidebar is collapsed, once changes persist, you can look at margin and adjust accordingly.

Reply:
Also the arrow placement should be 80% down and should overlap with sidebar and interface section in the middle vertically